site/gatsby-site/src/components/cite/RemoveDuplicateModal.js

```diff
@@ -0,0 +1,194 @@
+import React from 'react';
+import { useMutation, useQuery } from '@apollo/client/react/hooks';
+import { FIND_INCIDENT, UPDATE_INCIDENT } from '../../graphql/incidents';
+import { INSERT_DUPLICATE } from '../../graphql/duplicates';
+import { UPSERT_CLASSIFICATION, FIND_CLASSIFICATION } from '../../graphql/classifications';
+import { UPSERT_SUBSCRIPTION, FIND_FULL_SUBSCRIPTIONS } from '../../graphql/subscriptions';
+import IncidentsField from 'components/incidents/IncidentsField';
+import { Formik, Form } from 'formik';
+import { Button, Modal } from 'flowbite-react';
+import { Trans } from 'react-i18next';
+import useToast, { SEVERITY } from '../../hooks/useToast';
+
+export default function RemoveDuplicateModal({ incident, show, onClose }) {
+  const addToast = useToast();
+
+  const [updateIncident] = useMutation(UPDATE_INCIDENT);
+
+  const [insertDuplicate] = useMutation(INSERT_DUPLICATE);
+
+  const [updateClassification] = useMutation(UPSERT_CLASSIFICATION);
+
+  const [updateSubscription] = useMutation(UPSERT_SUBSCRIPTION);
+
+  const { data: classificationsData, loading: classificationsLoading } = useQuery(
+    FIND_CLASSIFICATION,
+    { variables: { query: { incidents: { incident_id: incident.incident_id } } } }
+  );
+
+  const { data: subscriptionsData, loading: subscriptionsLoading } = useQuery(
+    FIND_FULL_SUBSCRIPTIONS,
+    { variables: { query: { incident_id: { incident_id: incident.incident_id } } } }
+  );
+
+  return (
+    <Modal {...{ show, onClose }}>
+      <Modal.Header>
+        <Trans>Remove Duplicate</Trans>
+      </Modal.Header>
+      <Modal.Body>
+        <Formik initialValues={{ duplicateIncidentId: [] }}>
+          {({ values, isSubmitting, setSubmitting }) => {
+            const { data: duplicateIncidentData, loading: duplicateIncidentLoading } = useQuery(
+              FIND_INCIDENT,
+              { variables: { query: { incident_id: values.duplicateIncidentId?.[0] } } }
+            );
+
+            const submitBlocked =
+              duplicateIncidentLoading ||
+              !duplicateIncidentData ||
+              !values.duplicateIncidentId ||
+              values.duplicateIncidentId.length == 0 ||
+              classificationsLoading ||
+              !classificationsData ||
+              !subscriptionsData ||
+              subscriptionsLoading ||
+              isSubmitting;
+
+            const submit = async () => {
+              setSubmitting(true);
+              const reportIds = removeTypename(
```

Comment:
We don't need to call `removeTypename` function here since we are using just the `report_number`